lms/djangoapps/course_api/blocks/serializers.py

```diff
@@ -8,6 +8,25 @@
 from .transformers import SUPPORTED_FIELDS
 
 
+# This lists the names of all fields that are allowed
+# to be show to users who do not have access to a particular piece
+# of content
+AUTHORIZATION_DENIED_FIELDS = [
```

Comment:
If these fields are allowed, having the name include "denied" is confusing.

Reply:
Hm, true.